rollup/conf/config.json

```diff
@@ -66,6 +66,7 @@
       "l1_commit_gas_limit_multiplier": 1.2
     },
     "chunk_proposer_config": {
+      "enable_conditional_compress": false,
```

Comment:
why need an option to control it?  Isn't upgrading to codev4 by default?

Reply:
so that we can deploy rollup-relayer for backward compatibility first (if later PRs are to be deployed first), then reenable this config after running stable.

Reply:
I don't like this way, maybe we need a fork name and decide the hard fork time.

Reply:
makes sense. will remove it and discuss a formal method.

Reply:
removed. and use `DarwinV2` in this commit: [8fead44](https://github.com/scroll-tech/scroll/pull/1484/commits/8fead4470fe44ca447b35967a3e9f14de9b916b2).